docs/changelog.rst

```diff
@@ -6,6 +6,7 @@ v0.3.0 <2022-xx-xx>
 
 Newer numpy  (>v1.20.0) is required to avoid ABI compatibility issues. Please check the updated installation guide.
 
+* `#40`_: Introduce marine for Japanese accent estimation. Note that there could be a breakpoint regarding `run_frontend` because this PR changed the behavior of the API.
```

Comment:
Please add a link at the bottom of this file like:
```
.. _#40: https://github.com/r9y9/pyopenjtalk/pull/40
```

Reply:
@r9y9 
I added the link as you suggested! 👍 
Thank you!